front-end/src/services/utils.js

```diff
@@ -0,0 +1,20 @@
+export const utilFunc = {
+    stringifyUrl: (type, value) => {
+
```

Comment:
Can the whole function be simplified to this?
```suggestion
 return `&${type}=${value}
```
Done this way it won't need modifications if you want to add new types in the future 😃